src/core-services/cart/xforms/xformCartItems.js

```diff
@@ -76,26 +75,13 @@ async function xformCartItem(context, catalogItems, products, cartItem) {
  */
 export default async function xformCartItems(context, items) {
   const { collections, getFunctionsOfType } = context;
-  const { Catalog, Products } = collections;
+  const { Catalog } = collections;
 
   const productIds = items.map((item) => item.productId);
 
-  const catalogItems = await Catalog.find({
-    "product.productId": {
-      $in: productIds
-    },
-    "product.isVisible": true,
-    "product.isDeleted": { $ne: true },
-    "isDeleted": { $ne: true }
-  }).toArray();
+  const catalogItems = await Catalog.find({ "product.productId": { $in: productIds } }).toArray();
 
-  const products = await Products.find({
-    ancestors: {
-      $in: productIds
-    }
-  }).toArray();
```

Comment:
`products` was no longer being used by anything so we can eliminate a db lookup here